src/backend/distributed/master/worker_node_manager.c

```diff
@@ -414,6 +430,30 @@ ReferenceTablePlacementNodeList(LOCKMODE lockMode)
 }
 
 
+/*
+ * CoordinatorNode returns the WorkerNode object for coordinator node if it is
+ * added to pg_dist_node, otherwise errors out.
+ */
+WorkerNode *
+CoordinatorNode()
+{
+	if (!CoordinatorAddedAsWorkerNode())
+	{
+		ereport(ERROR, (errmsg("couldn't find coordinator node as it is not "
```

Comment:
this error message sounds a little confusing. Can't we avoid mentioning `pg_dist_node` in the message? `couldn't find the coordinator node in the metadata` or such

Reply:
https://github.com/citusdata/citus/pull/3852#discussion_r435094753 -> As we rename the objects on the table too, can `ConvertLocalTableToShard` be a better name ?

Reply:
https://github.com/citusdata/citus/pull/3852#discussion_r435083073 -> What about https://github.com/citusdata/citus/pull/3852/files#diff-35f1f08e94547e2b92f8f7af65c2fb5eR100 ?

Reply:
> this error message sounds a little confusing. Can't we avoid mentioning pg_dist_node in the message? couldn't find the coordinator node in the metadata or such

What about ? :
```c
ereport(ERROR, (errmsg("couldn't find the coordinator node in the metadata "
                                       "as it is not added as a worker")));
```